src/SFA.DAS.ApprenticeFeedback.Data/Configuration/FeedbackTransactionConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.ApprenticeFeedback.Domain.Entities;
+
+
+namespace SFA.DAS.ApprenticeFeedback.Data.Configuration
+{
+    public class FeedbackTransactionConfiguration : IEntityTypeConfiguration<FeedbackTransaction>
+    {
+        public void Configure(EntityTypeBuilder<FeedbackTransaction> builder)
+        {
+            builder.ToTable("FeedbackTransaction")
```

Comment:
Think it may be useful to add in the relationship between the transaction and the Apprentice Feedback Target as you will need both bits of information later.

Reply:
Added relationship